.travis.yml

```diff
@@ -8,3 +8,4 @@ before_install:
 
 script:
   - godep go test  ./pkg/... -race -cover
+  - godep go test  ./cmd/agent -test.run TestDir
```

Comment:
it is only a test case in cmd/agent/proc_test.go, because agent_test.go depend on zk, so i use TestDir only to check build error.
Is there any suggestion to only run one test file?
Or modify agent_test.go by using fake zk?